src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -1,7 +1,7 @@
 package core.basesyntax.dao;
 
 import core.basesyntax.db.Storage;
-import core.basesyntax.model.User;
+import core.basesyntax.service.model.User;
 
 public class StorageDaoImpl implements StorageDao {
     private static Long index = 0L;
```

Comment:
The index variable should be encapsulated properly to prevent external modification. Consider using a private method to handle the increment operation.